router/route/user.go

```diff
@@ -121,15 +122,104 @@ func retrieveUser(w http.ResponseWriter, r *http.Request) (int, error) {
 func retrieveUserScore(w http.ResponseWriter, r *http.Request) (code int, err error) {
 	username := mux.Vars(r)["username"]
 	score, err := model.GetUserScore(username)
+	err = json.NewEncoder(w).Encode(score)
 	if err != nil {
 		return http.StatusInternalServerError, causerr.New(err, "")
 	}
+	return http.StatusOK, nil
+}
 
-	err = json.NewEncoder(w).Encode(score)
```

Comment:
Yes, this was one weird merge, please check against master how it should look.

Reply:
I don't get it, I did exactly as usual. It automerged almost all files except 4 but the ones that it could not merge was kinda wierd.